.github/workflows/ci.yaml

```diff
@@ -23,7 +23,7 @@ env:
   IMAGE_NAME: ${{ vars.IMAGE_NAME || 'ramen' }}
   OPERATOR_SUGGESTED_NAMESPACE: ${{ vars.OPERATOR_SUGGESTED_NAMESPACE || 'ramen-system' }}
   # Constants
-  GO_VERSION: "1.21"
+  GO_VERSION: "1.22"
```

Comment:
Squashing this commit to to the first one would be best.

Reply:
But - there is a better way - instead of getting go version from this variable, which needs future maintenance, we can get it the go version from the go.mod file[1]

In each usage of setup-go:

      - name: Setup go
        uses: actions/setup-go@v5
        with:
          go-version-file: go.mod

[1] https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file

Reply:
Squashed. The proposed changed can be made in another PR, it's not straigh-forward, GO_VERSION constant is in use afterwards.